database/factories/MismatchFactory.php

```diff
@@ -23,13 +37,44 @@ public function definition()
     {
         return [
             'statement_guid' => 'Q' . $this->faker->randomNumber() . '$' . $this->faker->uuid(),
-            'property_id' => 'P' . $this->faker->randomNumber(),
-            'wikidata_value' => $this->getRandomValue(),
+            'property_id' => $this->faker->randomElement(array_keys(self::PROPERTIES)),
+            'wikidata_value' => function (array $attributes) {
+                $propertyId = $attributes['property_id'];
+                if (!array_key_exists($propertyId, self::PROPERTIES)) {
+                    throw new ValueError("Unknown property {$propertyId}, " .
+                        'you have to specify an explicit wikidata_value');
```

Comment:
This should never happen, because in line 40 we're exactly selecting a key from `self::PROPERTIES`, right?

Reply:
It might happen if a developer tries to create a mismatch with a custom property ID:

```php
$mismatch = Mismatch::factory()
    ->for($import)
    ->create(['property_id' => 'P12345']);
```

Reply:
Ok, so reading the source code of the `Factory` parent class, it seems the return value of the definition function is used as the basis for a reducer, that I guess somehow play into those `create` calls?